project/src/main/webapp/events-feed.css

```diff
@@ -50,7 +54,7 @@ html {
 }
 
 #event-container {
```

Comment:
here In my PR #238  have separated CSS into #event-container .card and  #search-container .card as well. Should be good.